e2e/cypress/tests/integration/enterprise/cloud/billing/pricing_modal_spec.js

```diff
@@ -54,7 +54,7 @@ describe('Pricing modal', () => {
         cy.apiRequireLicenseForFeature('Cloud');
     });
 
-    it('should not show Upgrade button in global header for non admin users', () => {
+    it('should show Upgrade button in global header for non admin users', () => {
```

Comment:
What tests could we add to test the new post notification?

Reply:
I have added cypress test to send the ping and also check admin DMs for the system-bot ping

Reply:
@AGMETEOR I logged in as a regular user and I don't see the upgrade button on the test server of this PR. @annelieseklein is the expectation that the Upgrade button is available for end users? It doesn't seem like it in the designs. 

The ability to see the pricing modal and ask for upgrade will be from the link @pvev will be adding on the main menu.

Reply:
For this PR, the Upgrade. button shows

Reply:
To not show the Upgrade button for end users - I believe is the correct behavior. However, not sure how the tests below are passing though 🤔

Reply:
I see what you mean. Let me try recreating this spinwick

Reply:
I think something is up with the spinwick. The logic to show upgrade button nolonger requires an admin

Reply:
Was that changed in this PR?

Reply:
On second thought, it could be because the subscription info is still limited to only admin. Let me check

Reply:
This https://github.com/mattermost/mattermost-server/pull/20540 and https://github.com/mattermost/mattermost-server/pull/20534 will fix once they're merged